### DISCUSSION TITLE
~

### DISCUSSION THREAD

**Discussion by kxzo** [vercel/vercel#8945] [Help]

**Comment by amyegan** [comment]
Hi @einKnuffy! Do you have any differences in environment variables between production and development?

**Reply by kxzo** [reply]
> Hi @einKnuffy! Do you have any differences in environment variables between production and development?

Nope, and that issue suddenly appeared. Just didn't touch the project for like a week and all packages are up to date.

**Reply by amyegan** [reply]
Can you share links to the preview and production deployments that show this difference?
